src/vstart.sh

```diff
@@ -564,13 +564,15 @@ EOF
 	    fi
 
 	    uuid=`uuidgen`
-	    echo "add osd$osd $uuid"
-	    $SUDO $CEPH_ADM osd create $uuid
+            echo "create osd with uuid ${uuid}"
```

Comment:
the [osd.$osd] section above is going to be wrong. What about allocating X osd ID and for osd in $(allocate CEPH_NUM_OSD) above so osd consistently reflect what it really is, even in the directory names ?